include/hwinfo/utils/stringutils.h

```diff
@@ -153,12 +153,29 @@ inline std::string wstring_to_string() { return ""; }
 inline std::string wstring_to_std_string(const std::wstring& ws) {
   std::string str_locale = setlocale(LC_ALL, "");
   const wchar_t* wch_src = ws.c_str();
+
+#ifdef _MSC_VER
```

Comment:
What issue did you face here, when compiling with MSVC?

Reply:
The issue is that the function `wcstombs` is marked deprecated in MSVC because it is unsafe. It is recommended to use the safe alternative, `wcstombs_s`, instead. However, the latter is only available on MSVC, not gcc.

Reply:
Thanks for the clarification!